product/views.py

```diff
@@ -1,3 +1,83 @@
-from django.shortcuts import render
+import json
+import jwt
+
+from django.views     import View
+from django.http      import JsonResponse
+
+from user.models      import (
+        UserTier,
+        UserType,
+        User
+        )
+from .models          import (
+        Brand,
+        Category,
+        Gender,
+        ClassLevel,
+        ProductImage,
+        Product,
+        ProductAge,
+        ProductUserlike,
+        Age
+        )
```

Comment:
```suggestion
from user.models      import (
    UserTier,
    UserType,
    User
)
from .models          import (
    Brand,
    Category,
    Gender,
        ClassLevel,
        ProductImage,
        Product,
        ProductAge,
        ProductUserlike,
        Age
        )
```

공백은 한 탭이면 충분할 것 같습니다 :)

Reply:
넵 수정했습니다.